Clojure tools support (#22)
gfredericks/how-to-ns: README.md

lein how-to-ns fix
```

### Clojure Tools

The official Clojure CLI supports installation of thirdparty [tools][].
To install cljfmt as a tool, run:

```bash
clj -Ttools install com.gfredericks/how-to-ns '{:mvn/version "0.2.11"}' :as how-to-ns
```

To use the tool to check for formatting errors in your project, run:

```bash
clj -Thow-to-ns com.gfredericks.how-to-ns.main/check
;; or with options
clj -Thow-to-ns com.gfredericks.how-to-ns.main/check '{:paths ["src" "test"] :require-docstring? false}'
```

And to fix those errors:

```bash
clj -Thow-to-ns com.gfredericks.how-to-ns.main/fix
;; or with options
clj -Thow-to-ns com.gfredericks.how-to-ns.main/fix '{:paths ["src" "test"] :require-docstring? false}'
```

> **NOTE!** If `:paths` is not specifieduse `src` and `test` folders in your project. 
> Other default explained in Customization section.

Reviewer: ```suggestion
**NOTE!** The default value for `:paths` is `["src" "test"]`.
Other defaults are explained in the Customization section below.
```

Author: I am very grateful to you for finding the opportunity to improve my wording. I am not a native speaker, so any help in this direction is very valuable to me.
